src/spikeinterface/core/template.py

```diff
@@ -31,7 +31,7 @@ class Templates:
         Array of unit IDs. If `None`, defaults to an array of increasing integers.
     probe: Probe, default: None
         A `probeinterface.Probe` object
-    is_scaled : bool, optional default: True
+    is_in_uV : bool, optional default: True
```

Comment:
Let's make sure that in the `Templates.from_zarr_group` we check for the existence of `is_scaled` for back-compatibility (e.g. with hybrid library)